test/smoke/package/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -254,6 +260,28 @@ export class SetupEnvironmentHelper {
         await IosSimulatorHelper.shutdownSimulator(device);
     }
 
+    public static terminateMacOSapp(appName: string) {
+        console.log(`*** Searching for ${appName} macOS application process`);
+        const searchForMacOSappProcessCommand = `ps -ax | grep ${appName}`;
```

Comment:
Please add the example of outputs to parse in comments, this helps to understand what is parsed using these commands and RegExps.